runtime/lua/vim/lsp/_watchfiles.lua

```diff
@@ -243,7 +243,7 @@ function M.register(reg, ctx)
           local last_type = change_cache[client_id][change.uri]
           if last_type ~= change.type then
             change_queues[client_id] = change_queues[client_id] or {}
-            table.insert(change_queues[client_id], change)
+            change_queues[client_id][#change_queues[client_id] + 1] = change
```

Comment:
Another instance where `table.insert` is easier to read.

Reply:
about there i think it's fine .  some single call i had revert .